src/components/forms/aggregate/__test__/AggregateForm.test.tsx

```diff
@@ -105,7 +105,7 @@ describe("AggregateForm MODIFY", () => {
     const mockAggregate = aggregate;
 
     it("renders with input value defined", async () => { 
-        await act(() => { renderWithProviders(<ConfirmationProvider><AggregateForm isCreate={false} aggregate={mockAggregate} usagePlans={usage_plan_list.items} /></ConfirmationProvider>)});
+        await act(() => { renderWithProviders(<ConfirmationProvider><AggregateForm isCreate={false} isUserWriter={true} aggregate={mockAggregate} usagePlans={usage_plan_list.items} /></ConfirmationProvider>)});
```

Comment:
Please add a test regarding the `isUserWriter={false}` scenario, testing that the fields are disabled and the save/delete button do not appear.